.github/dependabot.yml

```
@@ -0,0 +1,96 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
```

```suggestion
  open-pull-requests-limit: 10
  allow:
      - dependency-type: production
```

With this, we can move stuff we don't care about into `devDependencies` (e.g. typedoc) and stuff we do care about into `dependencies` (probably typescript etc). That should cut update spam down a lot.

Should generally go through `package.json` and see if there's anything we can cut. #3498 could get rid of a few.

Thanks for the tip @blueyed